src/script/cpp_api/s_security.cpp

```diff
@@ -232,19 +232,26 @@ void ScriptApiSecurity::initializeSecurityClient()
 		"collectgarbage",
 		"DIR_DELIM",
 		"error",
-		"getfenv",
+		// getfenv can be used to escape the sandbox
```

Comment:
Generally, Minetest doesn't use a function table sandbox so just because that wiki page says so doesn't mean it's actually possible. Or did you confirm that this is possible?

At the very least a comment should point to the source for the claims and be clear that the functions are _suspected_ to allow escaping the sandbox.

Reply:
OK, just tried it using the following script, and I couldn't manage to do it:

```c++
local ie = getfenv(string.sub)

local file = ie._G.io.open("/etc/passwd", "r")
local content = file:read("*all")
file:close()

print(content)
```

I guess setting a default environment also sets it for things like string.sub?

Reply:
I'd rather err on the side of caution and remove this